gcloud/storage/api.py

```diff
@@ -143,6 +135,9 @@ def create_bucket(bucket_name, project=None, connection=None):
 
     This implements "storage.buckets.insert".
 
+    If the bucket already exists, will raise
+    :class:`gcloud.exceptions.Conflict`.
+
```

Comment:
My feelings aren't strong on `:raises:`.  I do wish we could somehow document what exceptions we would be raising (whether generated locally or via an API call).  Unfortunately, the API docs don't specify the possible error response codes for each API call, so we can't just point to them.